docs/dev/DOCS.md

```diff
@@ -16,8 +16,16 @@ to be in `docs/index.rst`'s `toctree` though.
 
 ## Documentation Workflow
 
-To work with the docs, just edit reStructuredText or Markdown files in `docs`, 
-then run
+To work with the docs, just edit reStructuredText or Markdown files in `docs`.
+Before you compile the docs, you need to set the `ENVOY_VERSION` and 
+`GATEWAYAPI_VERSION` environment variables. For example:
+
+```bash
+export ENVOY_VERSION=1.23
```

Comment:
I do not think we need to manually export these env variables, these vars are computed (eg version -e) by code for now.

Reply:
Can you revert the changes in this file? Other changes LGTM. Thanks!

Reply:
Done